Test/Core/Components/RigidBodyTest.cs

```diff
@@ -1,21 +1,54 @@
-﻿using System;
```

Comment:
While removing unused usings is good, please keep using directives grouped and the groups separated with a single newline. `NUnit.Framework` is a different group than the `Duality` usings, which is a different one than the `System` usings.